chunky/src/java/se/llbit/chunky/world/ChunkPosition.java

```diff
@@ -43,13 +43,22 @@ public long getLong() {
 
   /**
    * @return The .mca name for the region with this position
+   *
+   * @deprecated Use {@link RegionPosition#getMcaName()}. Remove in 2.6
    */
+  @Deprecated
   public String getMcaName() {
     return String.format("r.%d.%d.mca", x, z);
   }
 
+  public RegionPosition asRegionPosition() {
+    return new RegionPosition(x >> 5, z >> 5);
+  }
+
   /**
    * @return The region position of this chunk position
+   *
+   * @deprecated Use {@link ChunkPosition#asRegionPosition()}. Remove in 2.6
    */
   public ChunkPosition getRegionPosition() {
```

Comment:
After much discussion on Discord, I guess that a breaking change here is fine for 2.5